Source/Core/VideoCommon/VideoConfig.cpp

```diff
@@ -261,6 +261,11 @@ void VideoConfig::VerifyValidity()
       iStereoMode = 0;
     }
   }
+  if (!backend_info.bSupportsBBox && g_ActiveConfig.bBBoxEnable)
+  {
+    PanicAlertT("Your graphics card does not support bounding box emulation this game requires. "
+                "You will experience bugs and crashes. This is not a Dolphin bug.");
```

Comment:
The first sentence is grammatically incorrect. Also, the wording of this message makes little sense when bbox is enabled globally, and the last sentence is... I'm not quite sure, but putting it in this message seems strange to me.